tools/strip_hashes.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+
+import os
+import re
+import sys
+
+def main(args):
+    out_lines = []
+    for line in args:
+        search = re.search(r'^(\S*==\S*).*$', line)
+        if search:
+            out_lines.append(search.group(1))
+    return os.linesep.join(out_lines)
+
+if __name__ == '__main__':
+    print(main(sys.argv[1:]))
```

Comment:
This is requiring the user provide the file as CLI arguments to this script:
```
tools/strip_hashes.py argparse==1.4.0 --hash=sha256:c31647edb69fd3d465a847ea3157d37bed1f95f19760b11a47aa91c04b666314 ...
```
I think this is a little awkward and doesn't work with the `RUN sh -c 'cat dependency-requirements.txt | /opt/certbot/tools/strip_hashes.py > unhashed_requirements.txt'` line in the Dockerfile.

Given how this script is being used, maybe it should just take the filename to read from as its first and only CLI argument?

With that said, if you also want to give it the option to read from stdin, I don't think the unix fans of the world would complain.